pkg/api/v1/resource/helpers_test.go

```diff
@@ -216,6 +220,97 @@ func TestExtractResourceValue(t *testing.T) {
 	}
 }
 
+func TestPodRequestsAndLimits(t *testing.T) {
+
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodOverhead, true)()
+
+	cases := []struct {
+		pod              *v1.Pod
+		cName            string
+		expectedRequests v1.ResourceList
+		expectedLimits   v1.ResourceList
+	}{
+		{
+			cName:            "just-limit-no-overhead",
+			pod:              getOverheadPod("foo", "", "9", "", "", "", ""),
+			expectedRequests: v1.ResourceList{},
+			expectedLimits: v1.ResourceList{
+				v1.ResourceName(v1.ResourceCPU): resource.MustParse("9"),
+			},
+		},
+		{
+			cName: "just-overhead",
+			pod:   getOverheadPod("foo", "", "", "", "", "5", "5"),
+			expectedRequests: v1.ResourceList{
+				v1.ResourceName(v1.ResourceCPU):    resource.MustParse("5"),
+				v1.ResourceName(v1.ResourceMemory): resource.MustParse("5"),
+			},
+			expectedLimits: v1.ResourceList{},
+		}, /* Failing Test - TODO/help
```

Comment:
This is the test case where I'm seeing issues -- this is the only point in the tests where we are actually adding Resource Lists.  From looking at the Add function, it seems the quantities string is cleared out, and that's the delta I see for the failure.  

Is there a best-practice for testing equality of resource lists?